File: root/data/src/main/java/org/sc/data/model/TrailPreview.java
package org.sc.data.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.List;


@Data
@Builder
@AllArgsConstructor
public class TrailPreview {
    private String id;
    private String code;
    private TrailClassification classification;
    private List<PlaceRef> locations;
    private PlaceRef startPos;
    private PlaceRef finalPos;
    private FileDetails fileDetails;
    private boolean bikeData;
    private TrailStatus trailStatus;
}

File: backend/src/main/java/org/sc/data/entity/TrailPreview.kt
package org.sc.data.entity

import org.sc.common.rest.TrailClassification
import java.util.*


data class TrailPreview constructor(val code: String,
                                    val classification: TrailClassification,
                                    val startPos : Position,
                                    val finalPos : Position,
                                    val date : Date)